SparkPost/2web2ui
# Update storybook to the latest version 🚀

## What Changed

- The `devDependency` [@storybook/addon-actions](https://github.com/storybooks/storybook) was updated from `4.1.14` to `5.0.1`.
- The `devDependency` [@storybook/react](https://github.com/storybooks/storybook) was updated from `4.1.14` to `5.0.1`.

[Update to these versions instead 🚀](https://github.com/SparkPost/2web2ui/compare/master...SparkPost:greenkeeper%2Fmonorepo.storybook-20190308180234) 
 

<details>
<summary>Commits</summary>
<p>The new version differs by  commits.</p>
<p>See the <a href="https://urls.greenkeeper.io/storybooks/storybook/compare/ae7a4cf0720cb9d528163f9411ade78aa379708a...ae7a4cf0720cb9d528163f9411ade78aa379708a">full diff</a></p>
</details>

## How to Test?

```
$ npm run storybook
```

Open http://localhost:9001 and make sure it renders correctly.

<img width="1387" alt="Screen Shot 2019-03-12 at 11 40 49 PM" src="https://user-images.githubusercontent.com/1335605/54252205-5347df80-4520-11e9-82e4-b6cb823a67a1.png">
